docs/serving/samples/hello-world/helloworld-php/README.md

```diff
@@ -162,7 +162,7 @@ folder) you're ready to build and deploy the sample app.
      for your app.
    - Automatically scale your pods up and down (including to zero active pods).
 
-1. To find the URL for your service, use
+2. To find the URL for your service, use
 
 
 === "kubectl"
```

Comment:
For the numbering to work correctly, this tab formatting (ie `=== "kubectl"`) must be indented by 4 spaces. This is because it is part of the step, and so it must be indented by 4 spaces relative to its parent element (the step).

The command on line 170 - 174 needs to be indented by 8 spaces. This because it must be indented by an additional 4 spaces relative to its parent element (the tab).